### PR TITLE
[V4] Feat: choose fee level in confirm page

### PR DESCRIPTION
**Pull request by Gamboster**

### CODE REVIEW

File: src/pages/send/choose-fee-level/choose-fee-level.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            </div>
+            <div translate>Current fee rate for this policy</div>
+          </div>
+        </div>
+      </div>
+    </ion-card>
+
+    <ion-card class="fee-custom" *ngIf="customFeePerKB">
+      <ion-item>
+        <ion-label color="primary" floating>Enter custom fee in sat/byte</ion-label>
```
Comment (JDonadio) [inline src/pages/send/choose-fee-level/choose-fee-level.html]: Translate

File: src/pages/send/choose-fee-level/choose-fee-level.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        <ion-icon ios="ios-alert" md="md-alert"></ion-icon>
+        <span *ngIf="showMinWarning" translate>Your fee is lower than recommended.</span>
+        <span *ngIf="showMaxWarning" translate>You should not set a fee higher than {{maxFeeRecommended}} satoshis/byte.</span>
+      </ion-item>
+    </ion-card>
+  </div>
+
+  <ion-list [hidden]="!feeLevel">
+    <ion-item class="fee-level">
+      <ion-label>Fee level</ion-label>
```
Comment (JDonadio) [inline src/pages/send/choose-fee-level/choose-fee-level.html]: Translate

File: src/pages/send/choose-fee-level/choose-fee-level.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            <div translate>Current fee rate for this policy</div>
+          </div>
+        </div>
+      </div>
+    </ion-card>
+
+    <ion-card class="fee-custom" *ngIf="customFeePerKB">
+      <ion-item>
+        <ion-label color="primary" floating>Enter custom fee in sat/byte</ion-label>
+        <ion-input type="number" placeholder="{{'Enter custom fee'|translate}}" min="minFee" max="maxFee" (ionChange)="checkFees(customSatPerByte)"
```
Comment (JDonadio) [inline src/pages/send/choose-fee-level/choose-fee-level.html]: Should not allow negative numbers. Set min value > 0

File: src/pages/send/choose-fee-level/choose-fee-level.ts [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+      this.updateFeeRate();
+    }).catch((err: any) => {
+      this.loadingFee = false;
+      this.showErrorAndClose(null, err);
+      return;
+    });
+  }
+
+  public close() {
+    this.viewCtrl.dismiss({ newFeeLevel: "Hola mundo" });
```
Comment (JDonadio) [inline src/pages/send/choose-fee-level/choose-fee-level.ts]: That string should be removed

**Comment by gabrielbazan7** [comment]
use buttonText as label for "send button"

**Pull request merged**